test/e2e/support/bootstrap.js

```diff
@@ -3,10 +3,15 @@
  */
 import puppeteer from 'puppeteer';
 
-const { PUPPETEER_HEADLESS, PUPPETEER_SLOWMO } = process.env;
+/**
+ * Node dependencies
+ */
+import { visitAdmin } from './utils';
+
+const { PUPPETEER_HEADLESS, PUPPETEER_SLOWMO, PUPPETEER_TIMEOUT } = process.env;
 
 // The Jest timeout is increased because these tests are a bit slow
-jest.setTimeout( 100000 );
+jest.setTimeout( PUPPETEER_TIMEOUT || 100000 );
```

Comment:
This seemed a handy thing to have so I added it, but if it's no good let me know.